lib/src/hosts/uhst_host.dart

```diff
@@ -129,6 +130,18 @@ class UhstHost with HostSubsriptionsMixin implements UhstHostSocket {
     }
```

Comment:
the first thing we should do in the exception handler is to close the relay eventstream.

Reply:
@Arenukvern please don't forget this one

Reply:
working on it

Reply:
done

Reply:
I don't see it, should be here:
```
} on Exception catch (exception) {
```